src/index.js

```diff
@@ -109,853 +107,773 @@ const Sink = require('./sink.js');
  * region_tag:logging_quickstart
  * Full quickstart example:
  */
-function Logging(options) {
-  if (!(this instanceof Logging)) {
-    return new Logging(options);
-  }
-
-  // Determine what scopes are needed.
-  // It is the union of the scopes on all three clients.
-  let scopes = [];
-  let clientClasses = [
-    v2.ConfigServiceV2Client,
-    v2.LoggingServiceV2Client,
-    v2.MetricsServiceV2Client,
-  ];
-  for (let clientClass of clientClasses) {
-    for (let scope of clientClass.scopes) {
-      if (clientClasses.indexOf(scope) === -1) {
-        scopes.push(scope);
+class Logging {
+  constructor(options) {
+    if (!(this instanceof Logging)) {
+      return new Logging(options);
```

Comment:
I think this is causing the coverage loss: https://codecov.io/gh/googleapis/nodejs-logging/pull/219/diff?src=pr&el=tree#D8-113

Do we still need it?

Reply:
Great call.  Removed!